### PR TITLE
[RC2] Convert markdown to rst

### PR DESCRIPTION
**Pull request by collivier**
done by pandoc + manual edit in chapter02 (wrap table) and in
chapter04 (long url).

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_cert/RC2/chapters/chapter01.rst
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter01.rst:7]: What is this?

File: doc/ref_cert/RC2/index.rst
Comment (pgoyal01) [inline doc/ref_cert/RC2/index.rst:53]: Didn't generate the TOC

File: doc/ref_cert/RC2/chapters/chapter01.rst
Comment (collivier) [reply doc/ref_cert/RC2/chapters/chapter01.rst:7]: The image conversion.

File: doc/ref_cert/RC2/index.rst
Comment (collivier) [reply doc/ref_cert/RC2/index.rst:51]: yes it does when called by sphinx (you can simply call tox to install deps in a venv)

File: doc/ref_cert/RC2/chapters/chapter01.rst
Comment (CsatariGergely) [inline doc/ref_cert/RC2/chapters/chapter01.rst:4]: Didn't we agree on removing the bogometers?

File: doc/ref_cert/RC2/chapters/chapter02.rst
Comment (CsatariGergely) [inline doc/ref_cert/RC2/chapters/chapter02.rst:4]: Remove?

File: doc/ref_cert/RC2/chapters/chapter02.rst
Comment (CsatariGergely) [inline doc/ref_cert/RC2/chapters/chapter02.rst:85]: At the end we should have relative links here. Maybe this will require some work after we merged the conversion pr-s of other docs.

File: doc/ref_cert/RC2/conf.py
Comment (CsatariGergely) [inline doc/ref_cert/RC2/conf.py:2]: Anuket contributors

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_cert/RC2/chapters/chapter04.rst
Comment (CsatariGergely) [inline doc/ref_cert/RC2/chapters/chapter04.rst:45]: After all rst conversion is done we should have a relative link here. Until that maybe using the readthedocs link would be better.

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_cert/RC2/chapters/chapter02.rst
Comment (collivier) [reply doc/ref_cert/RC2/chapters/chapter02.rst:4]: ok - I do remove it in the orig md first.
trivial

File: doc/ref_cert/RC2/conf.py
Comment (collivier) [reply doc/ref_cert/RC2/conf.py:2]: It's usually the foundation or the first company contributing to it. Is that the reason of -1?!
I

**Review by collivier — COMMENTED** [review]
Sorry it's all about trivial change. I don't see the reason to block the PR. Is there issue I don't see in the content?

File: doc/ref_cert/RC2/chapters/chapter04.rst
Comment (collivier) [reply doc/ref_cert/RC2/chapters/chapter04.rst:45]: All readthedocs build fails. I don't see how it could be linked to a missing documentation ;)

File: doc/ref_cert/RC2/chapters/chapter02.rst
Comment (collivier) [reply doc/ref_cert/RC2/chapters/chapter02.rst:85]: No. At the end we would have internal links or intersphinx mappings. But we do find a smooth path as proposed here.

File: doc/ref_cert/RC2/chapters/chapter01.rst
Comment (collivier) [reply doc/ref_cert/RC2/chapters/chapter01.rst:4]: yes. But github doesn't help managing PR dependencies - I might fail in the // PR here.

File: doc/ref_cert/RC2/chapters/chapter04.rst
Comment (pgoyal01) [reply doc/ref_cert/RC2/chapters/chapter04.rst:45]: @collivier Until RA2 is converted can we keep the reference to the specific section 4.9? Once conversion is done we can refer to the specific section title.

File: doc/ref_cert/RC2/chapters/chapter02.rst
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.rst:85]: Since this chapter is referenced multiple times and also in case the chapters are reorganised and the numbering changes, can we define it externally as ".. _RA2-Ch-SIG-Reqts https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter06.md" and where referenced just use "RA2-Ch-SIG-Reqts_"?

Comment (collivier) [reply doc/ref_cert/RC2/chapters/chapter02.rst:80]: @pgoyal01 we won't link to a markdown file at the end. It will be done via ref as soon as all docs are in rst.

File: doc/ref_cert/RC2/index.rst
Comment (pgoyal01) [inline doc/ref_cert/RC2/index.rst:9]: does this info need to be updated

**Review by pgoyal01 — APPROVED** [review]

File: doc/ref_cert/RC2/index.rst
Comment (collivier) [reply doc/ref_cert/RC2/index.rst:9]: Yes well catch- thx . New changes as I do modify md first.

**Review by gkunz — APPROVED** [review]

**Comment by gkunz** [comment]
Hi @collivier . What is your plan regarding changing the absolute links you introduce here to rst references? Will that also happen before the next release? Absolute links break the release handling of RTD, i.e., by statically pointing to latest, we cannot switch between releases in RTD.

Alternatively, we can keep the existing relative links, which support releases and change those to references directly without the intermediate step of absolute links. I know that linkcheck does not support relative links, but changing them 2 times is potentially error prone.

**Comment by collivier** [comment]
@gkunz no the absolute links are updated per branch (see the patch vs stable/lakelse).
https://github.com/cntt-n/CNTT/pull/2754

My plan was to merge RST only after stable/lakelse was published and then to move to sphinx features once RM, RAX are updated to RST  (RCX depend on them).

Yes RTD is just a tmp fix waiting for RM and RAX supporting both RTD and Github rendering.

**Comment by collivier** [comment]
No, the relative links work only if you mix RST and MD - it will fail if you fully remove MD as done here (linkcheck quickly detects all of them).

**Comment by collivier** [comment]
Let's see if it works well in RTD (locally tested without any issue=

**Pull request merged**
